assets/list.js

```diff
@@ -0,0 +1,21 @@
+var now = new Date();
+// Filter to get upcoming list.
+var eventLists = document.getElementById('event-lists');
+var homeEvents = document.getElementById('home-events');
+// This is the home page.
+if (homeEvents != undefined) {
```

Comment:
I don't think the these checks are needed now: the jquery will only do something when the pattern matches.